internal/commands/import.go

```diff
@@ -40,11 +41,34 @@ func runImportCommand(wrapper wrappers.UploadsWrapper) func(cmd *cobra.Command,
 		if err != nil {
 			return err
 		}
+		projectName, err := cmd.Flags().GetString(commonParams.ProjectName)
+		if err != nil {
+			return err
+		}
 
 		if importFileType == "" || importFilePath == "" {
 			return errors.Errorf(clierrors.MissingImportFlags)
 		}
-		_, err = importFile(importFileType, importFilePath)
+		if projectName == "" {
+			return errors.Errorf(clierrors.ProjectNameIsRequired)
+		}
+
+		params := make(map[string]string)
+		params["name"] = projectName
+		resp, _, err := projectsWrapper.Get(params)
```

Comment:
and here. btw, all the project related - maybe you can export it to a private method

Reply:
Done under the projectWrapper